src/scheduler/logger.ts

```diff
@@ -0,0 +1,23 @@
+import { createLogger, format, transports } from "winston";
```

Comment:
scheduler specific logger.

you can `import { logger } ` or `import { parentLogger }`. Both are the same, use whichever is more convenient.